src/libs/ajax/DataRepo.ts

```diff
@@ -142,15 +180,46 @@ export interface JobModel {
   class_name?: string;
 }
 
+export interface SnapshotBuilderGetConceptsResponse {
+  result: SnapshotBuilderConcept[];
+}
+
+export interface SnapshotBuilderGetConceptHierarchyResponse {
+  result: SnapshotBuilderParentConcept[];
+}
+
+export interface SnapshotAccessRequestResponse {
```

Comment:
Reference `SnapshotAccessRequestResponse` from `jade-data-repo`, what are ways `UUID` has been implemented in the past?